back/engines/commercial/verification/app/services/verification/verification_service.rb

```diff
@@ -68,10 +68,8 @@ def verify_sync(user:, method_name:, verification_parameters:)
       response = method.verify_sync verification_parameters
       uid = response[:uid]
       user_attributes = response[:attributes] || {}
-      custom_field_values = response[:custom_field_values]&.stringify_keys || {}
-      user.update!(
-        **user_attributes,
-        custom_field_values: user.custom_field_values.merge(custom_field_values)
+      user.update_merging_custom_fields!(
+        user_attributes.merge(custom_field_values: response[:custom_field_values] || {})
```

Comment:
No longer need for the `stringify_keys`, why?

Reply:
`update_merging_custom_fields!` has it `attributes = attributes.deep_stringify_keys`